photon-server/src/main/java/org/photonvision/server/Server.java

```diff
@@ -67,7 +67,7 @@ private static void start(int port) {
 
                             // Increase the upload size limit (arbitrary, but need to be able to deal with large
                             // calibration JSONs)
-                            javalinConfig.http.maxRequestSize = (long) (50 * 1e6);
+                            javalinConfig.http.maxRequestSize = (long) (100 * 1e6);
```

Comment:
Looking at javalin docs. The purpose of this is to tell javalin what size data should it cache in the coproc and then give access to the request handler. We should instead be using an InputStream to handle the resource using `ctx. bodyAsInputStream()`. I think its worth looking into.

Reply:
Instead of ctx.body()?

Reply:
I believe so, you'd have to read the docs more on how to handle copying the file into tmp and then handling that but it should just return a file stream which we can trivially copy into a File object then save. I'd say handle all that in this PR.

Reply:
If we are handling this in the other PR, we should be fine to send this

Reply:
We are? Which other one?

Reply:
#1298 You might've checked out the wrong branch